priv/cabbage/apps/itest/test/itest/configuration_api_test.exs

```diff
@@ -13,10 +13,14 @@
 # limitations under the License.
 defmodule ConfigurationRetrievalTests do
   use Cabbage.Feature, async: true, file: "configuration_api.feature"
+  alias Itest.Reorg
   alias Itest.Transactions.Encoding
   require Logger
 
   setup_all do
+    Reorg.finish_reorg()
```

Comment:
if I'm not mistaking, every test in cabbage is executed in their own process. Perhaps it would be better if we tie Itest.Reorg to the test process, and make sure `Itest.Reorg` cleans up after himself in a `terminate/2` callback? If Itest.Reorg is linked with the test process, the terminate/2 will be called when the test finishes, even if abruptly!

Reply:
I tried to add `Reorg.finish_reorg()` to` on_exit`. Reorgs started happening less consistently. Also, more tests started failing